### ISSUE TITLE
how to set replace_sampler_ddp`=False

### ISSUE THREAD

**Issue by zhangshen12356** [manujosephv/pytorch_tabular#90]
if i want to us `sampler = get_balanced_sampler(train['target'].values.ravel())` when train the model `tabular_model.fit(train=train, validation=val, train_sampler=sampler)`. And then i will get an error 

```You seem to have configured a sampler in your DataLoader. This will be replaced 'pytorch_lightning.utilities.exceptions.MisconfigurationException: You seem to have configured a sampler in your DataLoader. This will be replaced  by `DistributedSampler` since `replace_sampler_ddp` is True and you are using distributed training. Either remove the sampler from your DataLoader or set `replace_sampler_ddp`=False if you want to use your custom sampler```

**Comment by manujosephv** [comment]
I guess distributed training is not something I have tested while creating this. In the latest main branch, I have already enabled setting any PyTorch Lightning Trainer flag using a new argument in `TrainerConfig` : `trainer_kwargs`. You can try passing `replace_sampler_ddp=False` through the kwargs and see if it works.

**Comment by zhangshen12356** [comment]
> I guess distributed training is not something I have tested while creating this. In the latest main branch, I have already enabled setting any PyTorch Lightning Trainer flag using a new argument in `TrainerConfig` : `trainer_kwargs`. You can try passing `replace_sampler_ddp=False` through the kwargs and see if it works.

like this?
trainer_config = TrainerConfig(
        auto_lr_find=False, # Runs the LRFinder to automatically derive a learning rate
        batch_size=32,
        max_epochs=100,
        replace_sampler_ddp=False,
        gpus=[3,4] #index of the GPU to use. 0, means CPU
    )
it does not work.

**Comment by manujosephv** [comment]
No, like this. But in the latest main branch. This is not a feature I have pushed to pypi yet.
```python
trainer_config = TrainerConfig(
auto_lr_find=False, # Runs the LRFinder to automatically derive a learning rate
batch_size=32,
max_epochs=100,
gpus=[3,4], #index of the GPU to use. 0, means CPU
trainer_kwargs=dict(replace_sampler_ddp=False)
)
``

**Comment by zhangshen12356** [comment]
> No, like this. But in the latest main branch. This is not a feature I have pushed to pypi yet.
> 
> ```python
> trainer_config = TrainerConfig(
> auto_lr_find=False, # Runs the LRFinder to automatically derive a learning rate
> batch_size=32,
> max_epochs=100,
> gpus=[3,4], #index of the GPU to use. 0, means CPU
> trainer_kwargs=dict(replace_sampler_ddp=False)
> )
> ``
> ```

If i use the latest main branch, how to set ```replace_sampler_ddp=False````

**Comment by manujosephv** [comment]
You checkout the latest main branch, and install using `pip install .`

And then there is a new argument for `TrainerConfig` called `trainer_kwargs` with which you can pass any PyTorch Lightning Trainer argument that is not supported out of the box by the TrainerConfig. And the way you can do it is:

```python
trainer_config = TrainerConfig(
auto_lr_find=False, # Runs the LRFinder to automatically derive a learning rate
batch_size=32,
max_epochs=100,
gpus=[3,4], #index of the GPU to use. 0, means CPU
trainer_kwargs=dict(replace_sampler_ddp=False)
)
```
